TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithExecuteImmediateStatement.rdzSQL.txt

```diff
@@ -0,0 +1,23 @@
+﻿--- Diagnostics ---
+Line 7[14,20] <27, Error, Syntax> - Syntax error : An indicator variable must not be specified with a host variable in EXECUTE IMMEDIATE statement RuleStack=codeElement>executeImmediateStatement,  OffendingSymbol=[14,20:EXECUTE]<SQL_EXECUTE>
+Line 9[14,20] <27, Error, Syntax> - Syntax error : The constant must be Alphanumeric in EXECUTE IMMEDIATE statement RuleStack=codeElement>executeImmediateStatement,  OffendingSymbol=[14,20:EXECUTE]<SQL_EXECUTE>
```

Comment:
Instead of constant, use the word 'literal'. For example, 'Invalid literal found after EXECUTE IMMEDIATE, variable or string-expression was expected.'